docs-src/docs/guides/terraform_beyond_layer0.md

```diff
@@ -1,510 +0,0 @@
-# Deployment guide: Terraform beyond Layer0
```

Comment:
Is our plan to eventually rework this part of the guide by adding the missing Dockerrun et al resources to the guides repo? If so, we might want to consider just omitting the corresponding line in `mkdocs.yml` so the page isn't rendered instead of throwing out this entire file.

Reply:
I think this warrants more discussion because the example is a little contrived (a guestbook app that uses Consul?). I'm curious how practical such an example is. I'm wondering if there would be any unfortunate consequences by removing the line from the .yml file but leaving the markdown file hanging around. Mainly from a tidiness perspective.

Reply:
I think there would be some untidiness, but only in the sense that there's an unused file in the directory - not anything user-facing. I think the idea behind Seshi's "Terraform Beyond Layer0" example is just to illustrate some additional tips and best practices about Terraform that aren't intrinsic to the core walkthroughs. It's contrived in order to provide a user some familiarity with the application being deployed; a framework for the new ideas.

That said, I'm not actually applying pressure for this change - just wanted to bring it up as a thought.